addons/web/static/src/js/views/list/list_controller.js

```diff
@@ -504,6 +504,22 @@ var ListController = BasicController.extend({
             isDomainSelected: this.isDomainSelected,
         });
     },
+    _isValueSet(fieldType, value) {
+        switch (fieldType) {
+            case 'boolean':
+            case 'one2many':
+            case 'many2many':
+                return true;
+            case 'integer':
+            case 'monetary':
+            case 'float':
+                return typeof value === 'number';
```

Comment:
I think we could return true for numeric fields as well, because their value cannot be something else than a number